README.md

```diff
@@ -1151,6 +1151,15 @@ Defaults:
 - _schemas_: an array or object of schemas that will be added to the instance. In case you pass the array the schemas must have IDs in them. When the object is passed the method `addSchema(value, key)` will be called for each schema in this object.
 - _logger_: sets the logging method. Default is the global `console` object that should have methods `log`, `warn` and `error`. Option values:
   - custom logger - it should have methods `log`, `warn` and `error`. If any of these methods is missing an exception will be thrown.
+    ```javascript
+    var ajv = new AJV.default({
+      logger: {
+        log: log => console.log(log),
```

Comment:
All (or most) examples in the spec atm are ES5. While it’s worth revisiting, it should stay consistent, so let’s stick to ES5 please. You can create another issue to discuss moving examples to ES6 (with the comment the Ajv code is still ES5 for wider compatibility).